data/core.yaml

```diff
@@ -514,7 +514,9 @@ en:
       title: Cycle Highway Tag
       key: C
       description: Shortcut to cycle selected highway through several different tags.
-      annotation: Changed highway tag of selected way.
+      annotation: |
```

Comment:
Ah! There should be two annotations in separate entries, something akin to: 

```
highway_annotation: Changed highway tag of selected way
crosswalk_annotation: Changed crossing tags of selected way
```

Reply:
This will be similar for the code you will need to refactor, as the operation code currently assumes that the only operation it needs to do is cycle highway tags. You will need to refactor the code so that it can decide whether it should cycle tags on a highway or on a crosswalk.

Reply:
Thank you for the review, @Bonkles. I will implement the changes.